easybuild/tools/hooks.py

```diff
@@ -61,6 +61,7 @@
 
 START = 'start'
 PARSE = 'parse'
+POISED = 'poised'
```

Comment:
Not sure about `poised`, maybe we can find a better name (see also #4303)

Reply:
Just for future reference: as discussed in Slack: `build_and_install_loop` is clearer, so let's go with that (for lack of a better naming)